src/js/components/covid19/spendingOverTime/AmountTab.jsx

```diff
@@ -19,20 +21,20 @@ const propTypes = {
     setActiveTab: PropTypes.func.isRequired,
     active: PropTypes.bool,
     amount: PropTypes.number,
-    disabled: PropTypes.bool
+    disabled: PropTypes.bool,
+    dollarAmount: PropTypes.bool
 };
 
-
 const AmountTab = (props) => {
     const setActiveTab = () => {
         props.setActiveTab(props.type);
     };
 
     // Generate formatted amount string
-    let formattedAmount = formatMoney(props.amount);
+    let formattedAmount = props.dollarAmount ? formatMoney(props.amount) : formatNumber(props.amount);
```

Comment:
Would expect this to be handled in the model?

Reply:
In general, yes. But with these tabs being re-used all over the page, I think it will be less code / redundancy to handle formatting here rather than making new models for each section.